From 7a8c458b7acf4732af74317f8a535077eb451b1e Mon Sep 17 00:00:00 2001
From: Ming Liu <ming.liu@windriver.com>
Date: Thu, 17 Jul 2014 05:37:08 -0400
Subject: [PATCH] scl: fix wrong ownership during installation

The ownership of build user is preserved for some target files, fixed it by
adding --no-same-owner option to tar when extracting files.

Signed-off-by: Ming Liu <ming.liu@windriver.com>

Upstream-Status: Backport [9045908]

Signed-off-by: Yi Fan Yu <yifan.yu@windriver.com>
---
 scl/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scl/Makefile.am b/scl/Makefile.am
index 940a467..3c19e50 100644
--- a/scl/Makefile.am
+++ b/scl/Makefile.am
@@ -51,7 +51,7 @@ scl-install-data-local:
 		fi; \
 	done
 	$(mkinstalldirs) $(DESTDIR)/$(scldir)
-	(cd $(srcdir)/scl; tar cf - $(SCL_SUBDIRS)) | (cd $(DESTDIR)/$(scldir) && tar xf -)
+	(cd $(srcdir)/scl; tar cf - $(SCL_SUBDIRS)) | (cd $(DESTDIR)/$(scldir) && tar xf - --no-same-owner)
 	chmod -R u+rwX $(DESTDIR)/$(scldir)
 
 scl-uninstall-local:
